eng/common/scripts/Verify-Links.ps1

```diff
@@ -295,6 +295,10 @@ $checkedLinks = @{};
 $badLinks = @{};
 $pageUrisToCheck = new-object System.Collections.Queue
 foreach ($url in $urls) {
+  if (!$url) {
```

Comment:
Added empty url check as it occurs several times and throw exception at non-relative places.

Reply:
We should try and have the other script filter these out as I don't think this is a case we should try to handle here.

Reply:
The Urls parameters will finally get into this method from both scripts (we accept $urls parameter here, it gates all cases occurred). It is better to have null or empty checking before it ran into null pointer exception.

Reply:
I don't have an issue with having the check I just want to be sure it isn't needed in our normal usages. If we are passing empty or null values we should fix those.

Reply:
Do you want to throw error here whenever the case?

Reply:
No I'm fine with this being a warning I just want to make sure we don't ever hit this warning in our normal workflow.

Reply:
Right, I did not see we ran into this empty link before. Maybe we can remove the empty check here, and add prints out the url in NormalizedUrl function. It helps us figure out what's the url looks like when exception thrown.